doc/api/process.md

```diff
@@ -7,8 +7,7 @@
 <!-- source_link=lib/process.js -->
 
 The `process` object provides information about, and control over, the current
-Node.js process. While it is available as a global, it is recommended to
-explicitly access it via require or import:
+Node.js process.
```

Comment:
My preference here would be to add text explaining *why* the global should be avoided, but I don't know what that reason is. Also, with this change, there is now no mention of `process` being available as a global and that is something that should be mentioned.

Reply:
It's mentioned here: https://github.com/nodejs/node/blob/175638b7a412e8710e39cf2c69ef0b0039aa58af/doc/api/globals.md#L467-L477
I don't think we need more, especially if the intent is to promote importing/requiring the core module rather than rely on the global property.